fluent-bit/defaults.yaml

```diff
@@ -10,7 +10,6 @@ fluent_bit:
     flush: 5
     daemon: 'Off'
     log_level: info
-    parsers_file: parsers.conf
```

Comment:
why remove this? do we manually configure parsers somewhere or is fluent bit just a forwarder?

Reply:
This was causing the parsers to be written twice to the config and causing errors when fluent bit started up. I specify the parsers in the pillar file so this is not needed here.